src/RequestClientWrapper.ts

```diff
@@ -9,8 +9,8 @@ import { RequestConfig } from './model/RequestConfig';
 import { createClient } from '@supabase/supabase-js';
 
 const GENERIC_ERROR_CODE = -1;
-const SUPABASE_URL = '';
-const SUPABASE_ANON_KEY = '';
+const SUPABASE_URL = 'https://api.channelape.io/';
```

Comment:
@cduda is the plan not to be able to hit staging? It looks like we won't be able to set any other API host other than production? That's fine for now but might pose a problem in the future when we want to test beta webapp changes.

Reply:
@ryankazokas I wasn't sure how to go about it, so I hard-coded it. It wasn't the best decision, but I had spent a lot more time on that story than I wanted to.